airflow/providers/amazon/aws/operators/rds.py

```diff
@@ -314,13 +244,15 @@ def __init__(
         *,
         db_type: str,
         db_snapshot_identifier: str,
+        wait_for_completion: bool = True,
```

Comment:
This is a feature addition. A `db_snapshot_deleted` boto waiter exists but wasn't used.

Reply:
Nice! Love that! Could you add unit tests associated to this new feature?